.github/dependabot.yml

```diff
@@ -71,7 +71,9 @@ updates:
         update-types:
           - "version-update:semver-major"
           - "version-update:semver-minor"
-
+      - dependency-name: "com.github.johnrengelman.shadow"
+          update-types:
```

Comment:
The name is a bit confusing but `update-types` is a list of update types to ignore. So we would want `version-update:semver-major` here. See dependabot docs [here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore).

Reply:
Also, this tabbing does not look correct.